.github/workflows/reusable-ubuntu-ci.yml

```
@@ -105,6 +115,20 @@ jobs:
         with:
           api_token: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Get Fast CDR branch
+        id: get_fastcdr_branch
+        uses: eProsima/eProsima-CI/ubuntu/get_related_branch_from_repo@v0
+        with:
+          remote_repository: eProsima/Fast-CDR
+          fallback_branch: master
```

Using the input value
```suggestion
          fallback_branch: ${{ inputs.fastcdr-branch }}
```